lib/server.js

```diff
@@ -135,7 +129,10 @@ export default function main() {
                 workerPid: worker.process.pid,
             });
             setTimeout(() => {
-                if (!worker.isDead()) {
+                // TODO: When upgrading to Node v6,
+                // instead of !worker.suicide,
+                // use !worker.exitedAfterDisconnect
```

Comment:
worker.suicide was deprecated -- https://nodejs.org/dist/latest-v6.x/docs/api/cluster.html#cluster_worker_suicide

This was not intended to definitively fix the IPC issue.  There is likely a race condition in the node code, but we should only end up in this situation and see this error if there was another problem causing a crash.